openmicroscopy/bioformats
# ZeissCZIReader - Number Format Exception fix

Issue was raised in issue: https://github.com/openmicroscopy/bioformats/issues/2431
Sample QA filed provided is QA-17199

Null pointer occured when the position object is not null but the values of child are null. This resulted in a number format exception while attempting to parse the null value to integer.

I have put in place extra null pointer and isEmpty checks to deal with this scenario.

To reproduce:
- Attempt to load the file provided in QA-17199
- Verify that a Number Format Exception is thrown

To test:
- Verify that all tests and builds remain green
- Verify that the file provided in QA-17199 imports successfully without errors and displays as expected
